.dockerignore

```diff
@@ -0,0 +1,2 @@
+.git
+.gitignore
```

Comment:
Had a chat with @digininja He suggest that we can also ignore `README.md` and `fab.rb`

Reply:
Ok, I'm modifying it